readthedocs/core/unresolver.py

```diff
@@ -216,7 +246,7 @@ def _unresolve_path(self, parent_project, path, check_subprojects=True):
             if response:
                 return response
 
-        return None, None, None
+        return parent_project, None, None
```

Comment:
Why we are returning the `parent_project` now? What changed in the logic?

Reply:
If nothing matches, the current project is the parent project, this was being handled outside, but having this here makes it easy easier (no need to do `project_result or project`).